### PR TITLE
Enhancement/toolbar middle end events

### PR DESCRIPTION
**Pull request by PetarRing**

### CODE REVIEW

File: src/app/examples/toolbar-example/toolbar-example.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
-<kirby-toolbar title="Toolbar title" [themeColor]="color?.name" (back)="backButtonSelected()">
-  <kirby-icon secondary size="md"></kirby-icon>
-  <kirby-icon end size="md" name="more"></kirby-icon>
+<kirby-toolbar title="Toolbar title" [themeColor]="color?.name" (back)="backButtonSelected()" (primarySelected)="primarySelected()" (secondarySelected)="secondarySelected()">
```
Comment (jakobe) [inline src/app/examples/toolbar-example/toolbar-example.component.html]: Please use this naming for events and event handlers (see: https://angular.io/guide/styleguide#dont-prefix-output-properties):
```suggestion
<kirby-toolbar title="Toolbar title" [themeColor]="color?.name" (back)="onBack()" (primarySelect)="onPrimarySelect()" (secondarySelect)="onSecondarySelect()">
```

File: src/app/examples/toolbar-example/toolbar-example.component.tns.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <StackLayout>
   <kirby-toolbar title="primary" themeColor="primary" (back)="backButtonSelected()">
```
Comment (jakobe) [inline src/app/examples/toolbar-example/toolbar-example.component.tns.html]: No events in the tns example?

File: src/app/showcase/toolbar-showcase/toolbar-showcase.component.html [hunk 3]
```diff
@@ -30,4 +30,25 @@ <h4>Properties:</h4>
...
+      <th>Name</th>
+      <th>Description</th>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>back</code></td>
+        <td>Emits a click/tap event for the back navigation button</td>
+      </tr>
+      <tr>
+        <td><code>middleSelected</code></td>
```
Comment (jakobe) [inline src/app/showcase/toolbar-showcase/toolbar-showcase.component.html]: Update event names

File: src/app/showcase/toolbar-showcase/toolbar-showcase.component.html [hunk 4]
```diff
@@ -30,4 +30,25 @@ <h4>Properties:</h4>
...
+        <td>Emits a click/tap event for the middle element</td>
+      </tr>
+      <tr>
+        <td><code>endSelected</code></td>
```
Comment (jakobe) [inline src/app/showcase/toolbar-showcase/toolbar-showcase.component.html]: Update event names

File: src/kirby/components/toolbar/toolbar.component.tns.html [hunk 5]
```diff
@@ -11,15 +11,15 @@
...
-      <ng-content select="[secondary]"></ng-content>
+      <ng-content select="[primary]"></ng-content>
     </StackLayout>
   </ActionItem>
-  <ActionItem ios.position="right" >
+  <ActionItem ios.position="right" (tap)="onSecondarySelected()">
     <StackLayout orientation="horizontal">
       <Label text="" class="action-item-spacer-small"></Label>
-      <ng-content select="[end]]"></ng-content>
+      <ng-content select="[secondary]]"></ng-content>
```
Comment (jakobe) [inline src/kirby/components/toolbar/toolbar.component.tns.html]:
```suggestion
      <ng-content select="[secondary]"></ng-content>
```

File: src/kirby/components/toolbar/toolbar.component.ts [hunk 6]
```diff
@@ -19,6 +19,8 @@ export class ToolbarComponent implements OnInit {
   @Input() hideBackButton: boolean;
 
   @Output() back: EventEmitter<null> = new EventEmitter<null>();
```
Comment (jakobe) [inline src/kirby/components/toolbar/toolbar.component.ts]: Remove obsolete generic type:
```suggestion
  @Output() back: EventEmitter = new EventEmitter();
```

File: src/kirby/components/toolbar/toolbar.component.ts [hunk 7]
```diff
@@ -19,6 +19,8 @@ export class ToolbarComponent implements OnInit {
...
+  @Output() primarySelected: EventEmitter<null> = new EventEmitter<null>();
```
Comment (jakobe) [inline src/kirby/components/toolbar/toolbar.component.ts]: Remove obsolete generic type:
```suggestion
  @Output() primarySelected: EventEmitter = new EventEmitter();
```

File: src/kirby/components/toolbar/toolbar.component.ts [hunk 8]
```diff
@@ -19,6 +19,8 @@ export class ToolbarComponent implements OnInit {
...
+  @Output() secondarySelected: EventEmitter<null> = new EventEmitter<null>();
```
Comment (jakobe) [inline src/kirby/components/toolbar/toolbar.component.ts]: Remove obsolete generic type:
```suggestion
  @Output() secondarySelected: EventEmitter = new EventEmitter();
```

File: src/kirby/components/toolbar/toolbar.component.ts (hunk 6 above)
Comment (PetarRing) [reply src/kirby/components/toolbar/toolbar.component.ts]: Cannot remove generic type this way without provoking lint. Has to be: 
@Output() back = new EventEmitter();

Comment (jakobe) [reply src/kirby/components/toolbar/toolbar.component.ts]: Sorry my bad :)

**Pull request merged**